webgrid/filters.py

```diff
@@ -695,20 +711,22 @@ def process(self, value, is_value2):
 
 class DateTimeFilter(DateFilter):
     def format_display_vals(self):
-        if isinstance(self.value1, dt.datetime) and self.op in (
+        check_ops = (
             ops.eq.key,
             ops.not_eq.key,
             ops.less_than_equal.key,
             ops.greater_than_equal.key,
             ops.between.key,
             ops.not_between.key
-        ):
+        )
+        if isinstance(self.value1, dt.datetime) and self.op in check_ops:
             self.value1_set_with = self.value1.strftime('%m/%d/%Y %I:%M %p')
-        if isinstance(self.value2, dt.datetime) and self.op in (
-            ops.between.key,
-            ops.not_between.key
-        ):
+            if self.op in check_ops[:4] and self._has_date_only1:
```

Comment:
Instead of slicing here, could you split the list above into two named lists and then combine them with `check_ops = first_list + second_list`? This slicing seems particularly fragile.